src/java/org/openzal/zal/ProvisioningImp.java

```diff
@@ -40,66 +29,98 @@
 import com.zimbra.cs.ldap.ZSearchControls;
 import com.zimbra.cs.ldap.ZSearchScope;
 import com.zimbra.cs.ldap.unboundid.UBIDLdapContext;
+import com.zimbra.cs.mailbox.ACL;
+import com.zimbra.cs.mailbox.Contact;
+import com.zimbra.cs.mailbox.Folder;
+import com.zimbra.cs.mailbox.MailItem;
+import com.zimbra.cs.mailbox.MailServiceException;
+import com.zimbra.cs.mailbox.Mailbox;
 import com.zimbra.cs.util.ProxyPurgeUtil;
-import javax.annotation.Nonnull;
-
 import com.zimbra.soap.ZimbraSoapContext;
+import com.zimbra.soap.type.GalSearchType;
+import com.zimbra.soap.type.TargetBy;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.regex.Pattern;
 import org.dom4j.QName;
-import org.openzal.zal.exceptions.*;
+import org.openzal.zal.exceptions.ExceptionWrapper;
+import org.openzal.zal.exceptions.InvalidRequestException;
+import org.openzal.zal.exceptions.NoSuchAccountException;
+import org.openzal.zal.exceptions.NoSuchDistributionListException;
+import org.openzal.zal.exceptions.NoSuchDomainException;
+import org.openzal.zal.exceptions.NoSuchGrantException;
+import org.openzal.zal.exceptions.NoSuchGroupException;
+import org.openzal.zal.exceptions.NoSuchZimletException;
+import org.openzal.zal.exceptions.UnableToFindDistributionListException;
 import org.openzal.zal.exceptions.ZimbraException;
 import org.openzal.zal.lib.Filter;
+import org.openzal.zal.log.ZimbraLog;
+import org.openzal.zal.provisioning.DirectQueryFilterBuilder;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 
-import com.zimbra.cs.account.*;
-import com.zimbra.cs.account.accesscontrol.*;
-import com.zimbra.cs.gal.GalSearchControl;
-import com.zimbra.cs.gal.GalSearchParams;
-import com.zimbra.cs.gal.GalSearchResultCallback;
-import com.zimbra.common.soap.Element;
-import com.zimbra.common.account.Key;
-import com.zimbra.common.service.ServiceException;
-import com.zimbra.cs.ldap.ZLdapFilter;
-import com.zimbra.cs.ldap.ZLdapFilterFactory;
-import com.zimbra.soap.type.GalSearchType;
-import com.zimbra.soap.type.TargetBy;
 /* $if ZimbraVersion < 8.0.6 $
 import com.zimbra.common.account.Key.GranteeBy;
 /* $endif$ */
-
 /* $if ZimbraVersion >= 8.0.6 $*/
 import com.zimbra.soap.admin.type.GranteeSelector.GranteeBy;
 /* $endif$ */
 
-import com.zimbra.cs.mailbox.Contact;
-
-import javax.annotation.Nullable;
-import org.openzal.zal.log.ZimbraLog;
-import org.openzal.zal.provisioning.DirectQueryFilterBuilder;
-
 public class ProvisioningImp implements Provisioning
 {
   /* $if ZimbraVersion >= 8.7.0 $ */
-  public static String A_zimbraMaxAppSpecificPasswords                        = com.zimbra.cs.account.Provisioning.A_zimbraMaxAppSpecificPasswords;
```

Comment:
Avoid useless whitespaces changes

Reply:
Thank you for noticing it, I restored the initial import and formatting.